Report/create_reports.R

```diff
@@ -32,20 +32,30 @@ state_geos = locations %>%
                 filter(nchar(.data$geo_value) == 2) %>%
                 pull(.data$geo_value)
 signals = c("confirmed_incidence_num",
-            "deaths_incidence_num")
+            "deaths_incidence_num",
+            "confirmed_admissions_covid_1d")
 
 predictions_cards = get_covidhub_predictions(forecasters,
                                              signal = signals,
+                                             ahead = 1:28,
                                              geo_values = state_geos,
                                              verbose = TRUE,
-                                             use_disk = TRUE)
+                                             use_disk = TRUE) %>% 
+    filter(!(incidence_period == "epiweek" & ahead > 4))
+
 predictions_cards = predictions_cards %>%
-    filter(!is.na(predictions_cards$target_end_date))
-predictions_cards = predictions_cards %>% filter(target_end_date < today())
+    filter(!is.na(predictions_cards$target_end_date)) %>%
+    filter(target_end_date < today())
 
-# Only accept forecasts made Monday or earlier
+# For epiweek predictions, only accept forecasts made Monday or earlier.
+# target_end_date is the date of the last day (Saturday) in the epiweek
+# For daily predictions, accept any forecast where the target_end_date is later
```

Comment:
Is there a reason we aren't using the "Monday or earlier" cutoff for hospitalization data?

Reply:
The hospitalization forecasts are produced for every day following the forecast date; the target is `N day ahead inc hosp`. My understanding is that the "Monday or earlier" cutoff is only relevant for weekly forecasts, since we want to make sure that forecasts for a week aren't made with partial information for that week (i.e. it's easy to predict cases for a week if you know the values for 6 out of 7 days for that week). Will check with Dan.

Reply:
This approach matches Dan's understanding.